config-connector/solutions/projects/kpt/project-services/README.md

```diff
@@ -0,0 +1,42 @@
+Project Services
+==================================================
+
+# NAME
+
+  project-services
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to enable services on a project.
+
+## Requirements
+  Your service account must have roles/serviceusage.serviceUsageAdmin or roles/owner
+
+
+## Usage
+  First, set the project-id:
+
+  ```
+  kpt cfg set . project-id VALUE 
+  ```
+
+  Before applying a service, set the service name and the project-id annotation. For example, to enable Compute Engine:
+
+  ```
+  kpt cfg set . service-name compute.googleapis.com
+  ```
+
+  Note: the package-default value will enable PubSub.
+
+  To enable multiple services, copy the yaml for one service into a separate file and manually change the `metadata.name`.
+
+  Once your information is in the configs, simply apply.
+
+  ```
+  kubectl apply -f .
+  ```
```

Comment:
You may want to add an explanation about how the user will need to use `kubectl get gcpservices` rather than `kubectl get services` to see which services they've enabled with Config Connector.

Reply:
Done

Reply:
You seem to have reverted this in your latest commits.